src/icalendar/attr.py

```diff
@@ -1,24 +1,316 @@
+
+    EXDATE is defined in :rfc:`5545`.
+
+    Value Type:
+        The default value type for this property is DATE-TIME.
+        The value type can be set to DATE.
+
+    Property Parameters:
+        IANA, non-standard, value data type, and time
+        zone identifier property parameters can be specified on this
+        property.
+
+    Conformance:
+        This property can be specified in recurring "VEVENT",
+        "VTODO", and "VJOURNAL" calendar components as well as in the
+        "STANDARD" and "DAYLIGHT" sub-components of the "VTIMEZONE"
+        calendar component.
+
+    Description:
+        The exception dates, if specified, are used in
+        computing the recurrence set.  The recurrence set is the complete
+        set of recurrence instances for a calendar component.  The
+        recurrence set is generated by considering the initial "DTSTART"
+        property along with the "RRULE", "RDATE", and "EXDATE" properties
+        contained within the recurring component.  The "DTSTART" property
+        defines the first instance in the recurrence set.  The "DTSTART"
+        property value SHOULD match the pattern of the recurrence rule, if
+        specified.  The recurrence set generated with a "DTSTART" property
+        value that doesn't match the pattern of the rule is undefined.
+        The final recurrence set is generated by gathering all of the
+        start DATE-TIME values generated by any of the specified "RRULE"
+        and "RDATE" properties, and then excluding any start DATE-TIME
+        values specified by "EXDATE" properties.  This implies that start
+        DATE-TIME values specified by "EXDATE" properties take precedence
+        over those specified by inclusion properties (i.e., "RDATE" and
+        "RRULE").  When duplicate instances are generated by the "RRULE"
+        and "RDATE" properties, only one recurrence is considered.
+        Duplicate instances are ignored.
+
+        The "EXDATE" property can be used to exclude the value specified
+        in "DTSTART".  However, in such cases, the original "DTSTART" date
+        MUST still be maintained by the calendaring and scheduling system
+        because the original "DTSTART" value has inherent usage
+        dependencies by other properties such as the "RECURRENCE-ID".
+
+    .. note::
+
+        You cannot modify the EXDATE value by modifying the result.
+        Use :func:`icalendar.Component.add` to add values.
+
+    Example:
+
+        .. code-block:: pycon
+
+            >>> from icalendar import Event
+            >>> from datetime import datetime
+            >>> event = Event()
+
+            # Add a list of excluded dates
+            >>> event.add("EXDATE", [datetime(2025, 4, 28, 16, 5)])
+            >>> event.exdates
+            [datetime.datetime(2025, 4, 28, 16, 5)]
+
+    .. note::
+
+        If you want to compute recurrences, have a look at :ref:`Related Projects`.
```

Comment:
I think the note admonitions interfere with the proper rendering of Example in https://icalendar--814.org.readthedocs.build/en/814/api.html#icalendar.cal.TimezoneStandard.exdates, as well as the extra new line. This is one of five instances that have a similar issue, so it may be easier to do a global find and replace than have me use the GitHub review UI.

```suggestion
        .. note::

            You cannot modify the EXDATE value by modifying the result.
            Use :func:`icalendar.Component.add` to add values.

    Example:
        .. code-block:: pycon

            >>> from icalendar import Event
            >>> from datetime import datetime
            >>> event = Event()

            # Add a list of excluded dates
            >>> event.add("EXDATE", [datetime(2025, 4, 28, 16, 5)])
            >>> event.exdates
            [datetime.datetime(2025, 4, 28, 16, 5)]

        .. note::

            If you want to compute recurrences, have a look at :ref:`Related Projects`.
```

Reply:
Hm. It renders ok for me:

![grafik](https://github.com/user-attachments/assets/4ab7ca4b-edb9-4e70-a003-c4c5a6afdd38)

The examples is gray behind it but that might not matter much. I would say that is a styling issue and will be resolved over time. I don't mind as it is already better that before.

Reply:
It might look OK, but the admonitions are not in the correct context for semantic markup. Indentation matters.

Reply:
Right! In the sphinx markup, if does not work.

I added a description and I find the note at the end is enough.

![grafik](https://github.com/user-attachments/assets/7c983882-0dcf-4dc1-8fff-b74f525ae825)

The notes on exdate, rdate and rrule are now almost the same, too.